apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -1093,42 +1100,46 @@ export const AutoRepair = ({
 const ImageLoadingGeneric: React.FC<{
   title: string;
   children?: React.ReactNode | undefined;
-  top?: React.ReactNode | undefined;
-  bottom?: React.ReactNode | undefined;
   progress?: number;
-  backgroundPlaceholderText?: string;
-}> = ({
-  title,
-  top,
-  bottom,
-  children,
-  progress,
-  backgroundPlaceholderText,
-}) => {
+  lottieSource?: React.ComponentProps<
+    typeof FramedImageWithLottie
+  >["lottieSource"];
```

Comment:
Same: maybe we can export a `FramedImageWithLottieProps` to use here, to make it simpler to read